dice_ml/data_interfaces/public_data_interface.py

```diff
@@ -285,7 +293,7 @@ def get_decimal_precisions(self):
                     prec = len(str(modes[mx]).split('.')[1])
                     if prec > maxp:
                         maxp = prec
-                maxp = min(maxp, 4)
+                #maxp = min(maxp, 3)

```

Comment:
why is this commented out?

Reply:
This was present to evade a large search space while doing sparsity enhancement using linear search. For instance, if income is a decimal with precision 0.001 and ranges from 100 to 1000, then there are 900*10^3 moves, instead of 900 moves if income was an integer. Now, I removed this line since binary search option can handle them.